packages/app/src/settings/project/RecordKey.cy.tsx

```diff
@@ -20,11 +20,11 @@ describe('<RecordKey />', () => {
   })
 
   it('renders the record key view with the correct title', () => {
-    cy.findByText('Record Key')
+    cy.findByText('Record key')
   })
 
   it(`has an input that's hidden by default`, () => {
-    cy.get('code').as('Record Key Input')
+    cy.get('code').as('Record key input')
```

Comment:
```suggestion
    cy.get('code')
```
This alias wasn't being used, let's just remove it 😄